apps/functions/document-check/check-document/event-client.js

```diff
@@ -0,0 +1,56 @@
+import { getEventClient, EventType } from '@pins/event-client';
+import { getDocumentProperties, getDocumentFolders } from './back-office-api-client';
+import config from './config';
+const { serviceBus } = config;
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {string} guid
+ * @param {string} eventStatus
+ * @param {EventType} eventType
+ * */
+const handlePublishResult = async (context, guid, eventStatus, eventType) => {
+	const document = await getDocumentProperties(guid);
+	if (!document) {
+		throw new Error(`No document found with guid ${guid}`);
+	}
+
+	if (!document.fromFrontOffice) {
+		return;
+	}
+
+	const folders = await getDocumentFolders(guid);
+	if (!folders || folders.length < 2) {
+		throw new Error(
+			`Returned ${folders} when fetching folders for document from front office: "${guid}"`
+		);
+	}
+
+	const event = {
+		status: eventStatus,
+		deadline: folders[1].displayNameEn,
+		submissionType: folders[0].displayNameEn,
+		blobGuid: guid,
+		documentName: document.documentName
+	};
+
+	const eventClient = getEventClient(true, context.log, serviceBus.host);
+	await eventClient.sendEvents(serviceBus.topic, [event], eventType);
+};
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {string} guid
+ * */
+export const handleNotInfected = async (context, guid) =>
+	handlePublishResult(context, guid, 'SUCCESS', EventType.Success);
```

Comment:
Probably good to have these constants `SUCCESS` and `VIRUS_DETECTED` in some consts somewhere, so we know what they relate to.